CGPs/cgp-0095.md

```diff
@@ -0,0 +1,100 @@
+- $75k - Support, upgrades and maintenance over the first 12 months
+
+## Success Criteria
+As outlined in CGP-84, “Total Debt Value Locked” is a key metric by which the Celo community can measure the success of Centrifuge. Several companies in the rapidly expanding Credit Collective have registered interest in utilizing Centrifuge on Celo to launch their debt pools.
+
+Since Centrifuge enables the issuance of “ReFi” and Natural Capital assets (eg. carbon credit forward contracts, renewable energy infrastructure loans, traditional development finance) - cumulative impact metrics around CO2, gigawatts, sustainable development goals (SDGs), and domain-specific parameters are relevant indicators of success. 
+
+An important feature of this deployment is that Centrifuge pools on Celo can accept capital from Ethereum mainnet and other chains. Consequently, it is expected that launching Centrifuge and associated debt pools will drive significant bridging volume towards Celo. 
+
+## Known Risks
+Given that Centrifuge is a standalone web3 protocol, there is minimal risk in deploying the infrastructure itself on Celo. The primary risks are associated with the possibility of defaults on debt pools, which is not linked to web3 but rather the underlying RWAs. This risk has been historically mitigated by the diligence conducted by the Centrifuge governance community in the “[Pool Onboarding Process (POP)](https://gov.centrifuge.io/t/introducing-the-pool-onboarding-proposal-pop/3846)”.
+
+The Centrifuge community has recently established the [Credit Group](https://gov.centrifuge.io/t/introducing-the-centrifuge-credit-group/4994), a team of outside experts capable of providing risk analysis for Pools at launch. This function helps to ensure that the underlying deal that establishes the Pool is arranged in a way that is consistent with best practices, competitive to market standards, and most importantly, with due regard to appropriate risk management and compensation for investment returns. 
+
+# Implementation
+## Budget
+We request to set our spend permit on the community fund at 400,000 CELO - but hope to not use the full amount. This number accounts for CELO at $0.375 (150,000 cUSD from 400,000 CELO) and protects the budget from acute market volatility. Any unutilized CELO will remain in the community fund.
+## Multisig
+A Multisignature wallet hosted on Gnosis Safe will be used to manage and deploy the funds with a required 2 of 3 signature threshold. The contract address is 0x34CbB11f19D9B6F1fC54d20B5e248Db4a9B9afa2 and can be found [here](https://app.safe.global/home?safe=celo:0x34CbB11f19D9B6F1fC54d20B5e248Db4a9B9afa2). The signers on this multisig are:
+
+- Lucas Vogelsang (Centrifuge): 0xAAA0da6a9673110a15bA11fAc8eE142A8D86E117
+- Nirvaan Ranganathan (Flowcarbon / Climate Collective): 0x42B4a02f79438D558AB33E326638ed752b94D187
+- Tomer Bariach (Flori Ventures / Credit Collective): 0xEBe109b76b606C19DDC890fFcA3e992e6a0fB13D
+
```

Comment:
The multisign is missing this address:
Tomer Bariach (Flori Ventures / Credit Collective): 0xEBe109b76b606C19DDC890fFcA3e992e6a0fB13D

Reply:
Resolved on Safe